.github/workflows/python-lints.yml

```diff
@@ -8,7 +8,7 @@ jobs:
     strategy:
       max-parallel: 4
       matrix:
-        python-version: [3.6, 3.7, 3.8]
+        python-version: [3.7, 3.8, 3.9]
```

Comment:
From the [statuses of Python versions](https://devguide.python.org/versions/), Python 3.7 is already deprecated. Maybe we can use a more powerful test matrix with more recent Python versions (and also PyTorch versions?).
It's a more important change and it could be postponed to a future PR

Reply:
will be done in the next PR